src/annotations/AnnotationParser.php

```diff
@@ -228,6 +280,11 @@ public function parse($source, $path)
         $code = "return array(\n";
         $code .= "  '#namespace' => " . var_export($namespace, true) . ",\n";
         $code .= "  '#uses' => " . var_export($uses, true) . ",\n";
+
+        if (count($losingMethods)) {
+            $code .= "  '#losingMethods' => " . var_export($losingMethods, true) . ",\n";
```

Comment:
Do this unconditionally. When cache versioning would in place (not part of this commit), then the `isset` can be dropped in runtime checks for this array key.